### PR TITLE
Get sentence sorting feature done

### LINKED ISSUE #7: sort sentences alphabetically
using functionality from #2, #4, #5, sort the entered sentence

### PR DESCRIPTION
**Pull request by LoganParker**
Implement the sentence sorting feature, as well as fixed some spacing issues, and validated all test cases. Closes #7

### CODE REVIEW

File: preprocess.py [hunk 1]
```diff
@@ -52,16 +57,29 @@ def preproc_str(string: str, replaceContractions: bool = False) -> str:
...
     arr_str = preproc_str(string).split(" ")
-    return len(arr_str)
\ No newline at end of file
+    return len(arr_str)
+
+
+def sort_text(text: str) -> list:
+    """
+    This function sorts the inputted text string
+    :param text: A preprocessed string
```
Comment (vnajack) [inline preprocess.py:79]: for our project, we'll need to make sure we're consistent with function comments about parameters and returns! This looks good

File: preprocess.py [hunk 2]
```diff
@@ -1,48 +1,53 @@
...
     """
     return string.replace("\n", " ")
 
+
 def __remove_contractions(string: str) -> str:
     """
     Return a copy of the string with English contractions replaced.
     WARNING: will change word count.
     """
-    string = re.sub(r'\'s', ' is', string)
```
Comment (vnajack) [inline preprocess.py:25]: thank you for fixing the spacing!

**Review by edouarde1 — APPROVED** [review]
Keep in that all of the other functions return a string and this sort function returns a list.

This issue might not need adjustment as a sorted list of words may be more convenient.

File: test.py [hunk 3]
```diff
@@ -25,11 +27,21 @@ def test_word_count(self):
         test_string = "Our team had a debate for best names for looping variables ? i won"
         expected_count = 13
         self.assertEqual(expected_count, p.get_word_count(test_string))
+
     # test function to test equality of two value
     def test_wordsoup(self):
-        fname = "Read.txt"
+        fname = "text_files/read.txt"
```
Comment (vnajack) [inline test.py:33]: nice catch!

**Review by vnajack — APPROVED** [review]
Great work! Thank you for fixing the spacing in the files.

File: preprocess.py (hunk 1 above)
Comment (LoganParker) [reply preprocess.py:79]: Yes! With typescript we will need to be consistent as well, so it's important to note this

**Review by edouarde1 — COMMENTED** [review]
Its interesting how pycharm has different python standards than vscode

File: preprocess.py (hunk 2 above)
Comment (LoganParker) [reply preprocess.py:25]: Of course, ideally we will have a standardized linting process for the project :)

**Comment by LoganParker** [comment]
> Keep in that all of the other functions return a string and this sort function returns a list.
> 
> This issue might not need adjustment as a sorted list of words may be more convenient.

I think it just depends on what we want the function to do! Ideally our requirements for the project will be more concrete!

**Comment by LoganParker** [comment]
> Its interesting how pycharm has different python standards than vscode

or do you not have python linting enabled ;)

**Pull request merged**